rails-time-track/app/controllers/api/sessions_controller.rb

```diff
@@ -0,0 +1,25 @@
+module Api
+  class Api::SessionsController < ApplicationController
+    skip_before_action :require_login, only: :create
+
+    def create
+      if params.has_key?("email") && params.has_key?("password")
+        user = User.valid_login?(params[:email], params[:password])
+        if user
+          regenerate_and_signed_token(user)
+          render json: user
+        else
+          render_errors("Incorrect email or password", :bad_request)
+        end
+      else
+        render_errors("You have to pass the parameters 'email' and 'password'", :bad_request)
+      end
+    end
+
+    def destroy
+      current_user.invalidate_token
```

Comment:
Hi, can you explain me where did you define this variable?

Reply:
Of course, I created the instance variable in the application_controller